helm lint error around `.Capabilities.KubeVersion`

### Issue

`helm lint` gives the following error on a seemingly appropriate use of checking `.Capabilities.KubeVersion.Minor`:

```
 $ helm lint stable/spotify-docker-gc/
==> Linting stable/spotify-docker-gc/
...
[ERROR] templates/: render error in "spotify-docker-gc/templates/daemonset.yaml": template: spotify-docker-gc/templates/daemonset.yaml:6:25: executing "spotify-docker-gc/templates/daemonset.yaml" at <.Capabilities.KubeVe...>: can't evaluate field Minor in type *version.Info
```

### Details

The [chart](https://github.com/kubernetes/charts/pull/982) installs successfully and the template logic is evaluated as expected: yaml is not added if `KubeVersion.Minor` is less than 6, yaml is added if greater than or equal to 6.

The relevant template section looks like:
```
  {{- if ge .Capabilities.KubeVersion.Minor "6" }}
  updateStrategy:
    type: RollingUpdate
  {{- end }}
```

### Helm version
```
 $ helm version
Client: &version.Version{SemVer:"v2.4.1", GitCommit:"46d9ea82e2c925186e1fc620a8320ce1314cbb02", GitTreeState:"clean"}
Server: &version.Version{SemVer:"v2.4.1", GitCommit:"46d9ea82e2c925186e1fc620a8320ce1314cbb02", GitTreeState:"clean"}
```

--- a/pkg/lint/rules/template.go
+++ b/pkg/lint/rules/template.go
@@ -21,12 +21,15 @@ import (
 	"fmt"
 	"os"
 	"path/filepath"
+	"runtime"
 
 	"github.com/ghodss/yaml"
+	"k8s.io/apimachinery/pkg/version"
 	"k8s.io/helm/pkg/chartutil"
 	"k8s.io/helm/pkg/engine"
 	"k8s.io/helm/pkg/lint/support"
 	"k8s.io/helm/pkg/timeconv"
+	tversion "k8s.io/helm/pkg/version"
 )
 
 // Templates lints the templates in the Linter.
@@ -51,7 +54,17 @@ func Templates(linter *support.Linter) {
 	}
 
 	options := chartutil.ReleaseOptions{Name: "testRelease", Time: timeconv.Now(), Namespace: "testNamespace"}
-	caps := &chartutil.Capabilities{APIVersions: chartutil.DefaultVersionSet}
+	caps := &chartutil.Capabilities{
+		APIVersions: chartutil.DefaultVersionSet,
+		KubeVersion: &version.Info{
+			Major:     "1",
+			Minor:     "6",
+			GoVersion: runtime.Version(),
+			Compiler:  runtime.Compiler,
+			Platform:  fmt.Sprintf("%s/%s", runtime.GOOS, runtime.GOARCH),
+		},
+		TillerVersion: tversion.GetVersionProto(),
+	}
 	valuesToRender, err := chartutil.ToRenderValuesCaps(chart, chart.Values, options, caps)
 	if err != nil {
 		// FIXME: This seems to generate a duplicate, but I can't find where the first

--- a/pkg/lint/rules/testdata/albatross/templates/svc.yaml
+++ b/pkg/lint/rules/testdata/albatross/templates/svc.yaml
@@ -8,6 +8,8 @@ metadata:
     heritage: {{ .Release.Service | quote }}
     release: {{ .Release.Name | quote }}
     chart: "{{.Chart.Name}}-{{.Chart.Version}}"
+    kubeVersion: {{ .Capabilities.KubeVersion.Major }}
+    tillerVersion: {{ .Capabilities.TillerVersion }}
 spec:
   ports:
   - port: {{default 80 .Values.httpPort | quote}}

